### PR TITLE
Delete Feature - UI

### PR DESCRIPTION
**Pull request by amelendez37**

### CODE REVIEW

**Review by DChan0319 — APPROVED** [review]

**Review by jaba9531 — APPROVED** [review]
Looks Good

File: client/src/actions/index.js [hunk 1]
```diff
@@ -102,3 +103,20 @@ export const selectedWorkout = (workout) => {
...
 };
+
+export const deleteWorkout = async (workoutId, userId) => {
+  try {
+    await axios.delete(`http://localhost:8000/api/workouts/delete/${workoutId}`, {
+      headers: {
+        Authorization: `${document.cookie}`
+      }
+    });
+    const updatedWorkouts = await getWorkoutsList(userId);
```
Comment (zgallagher08) [inline client/src/actions/index.js]: you should just delete the one from the array and send it through this action to update the redux store instead

File: client/src/components/workoutsView/workoutsListItem.jsx [hunk 2]
```diff
@@ -1,38 +1,48 @@
...
-              :
-              <img className="float-right status" src={require('../../../dist/images/lock.png')}></img>
+            <img className="float-right status" src={require('../../../dist/images/earth.png')}></img>
+            :
+            <img className="float-right status" src={require('../../../dist/images/lock.png')}></img>
           }
+          <img className="float-left delete-icon" src={require('../../../dist/images/trash.png')} 
+           onClick={
+             async (e) => {
+               e.persist();
```
Comment (zgallagher08) [inline client/src/components/workoutsView/workoutsListItem.jsx]: what does e.persist do?

File: client/src/components/workoutsView/workoutsListItem.jsx [hunk 3]
```diff
@@ -1,38 +1,48 @@
...
+               await this.handleWorkoutClick.bind(this)();
```
Comment (zgallagher08) [inline client/src/components/workoutsView/workoutsListItem.jsx]: use **e.target.getAttribute('data')** to get the workout id, and you can send that to the action to delete the workout.

to add the data attribute to the trash can,

File: client/src/containers/workoutsListContainer.jsx [hunk 4]
```diff
@@ -24,21 +28,32 @@ class WorkoutsListContainer extends Component {
...
 
-  handleExerciseClick() { 
-    // 'this' refers to workoutListItem component
-    this.props.selectedWorkout(this.state.workout);
+  handleWorkoutClick(workout) { 
+    this.props.selectedWorkout(workout);
   }
   
+  toggleModal(e) {
+    e.stopPropagation();
```
Comment (zgallagher08) [inline client/src/containers/workoutsListContainer.jsx]: what does e.stopPropagation() do?

Comment (zgallagher08) [inline client/src/containers/workoutsListContainer.jsx:40]: this turned yellow, not sure why, but nevermind.

**Review by zgallagher08 — CHANGES_REQUESTED** [review]
Looks good, just a couple fixes

File: client/src/components/workoutsView/workoutModal.jsx [hunk 5]
```diff
@@ -12,11 +12,8 @@ class WorkoutModal extends Component {
           this.props.modalVisible ? 
             <div className="modal-workout">
               <div className="modal-workout-content">
-                <p>Delete {this.props.clickedWorkout.name}?</p>
-                <a 
-                 className="float-left modal-btn pointer"
-                 onClick={(e) => this.props.handleYesClick(e, this.props.clickedWorkout.id, this.props.userId)}
-                >Yes</a>
+                <p>Delete *get name*?</p>
```
Comment (zgallagher08) [inline client/src/components/workoutsView/workoutModal.jsx]: can you replace *get name* with the name?

File: client/src/components/workoutsView/workoutsListItem.jsx [hunk 6]
```diff
@@ -35,14 +35,7 @@ class WorkoutsListItem extends Component {
...
           }
-          <img className="float-left delete-icon" src={require('../../../dist/images/trash.png')} 
-           onClick={
-             async (e) => {
-               e.persist();
-               await this.handleWorkoutClick.bind(this)();
-               this.props.toggleModal(e);
-             }
-           }></img>
+          <img className="float-left delete-icon" src={require('../../../dist/images/trash.png')} data={this.props.workout.id} onClick={(e) => this.props.toggleModal(e)}></img>
```
Comment (zgallagher08) [inline client/src/components/workoutsView/workoutsListItem.jsx]: this is good, just for future reference:
onClick={(e) => this.props.toggleModal(e)}
is equivalent to
onClick={this.props.toggleModal}

**Review by zgallagher08 — APPROVED** [review]
Good, I left 2 comments (the first should be resolved, but not urgent)

**Review by zgallagher08 — APPROVED** [review]

**Pull request merged**